src/Nethermind/Nethermind.Specs.Test/OverridableReleaseSpec.cs

```diff
@@ -55,7 +55,7 @@ public OverridableReleaseSpec(IReleaseSpec spec)
 
         public bool IsEip155Enabled => _spec.IsEip155Enabled;
 
-        public bool IsEip158Enabled => _spec.IsEip158Enabled;
+        public bool IsEip158Enabled => _spec.IsEip1559Enabled;
```

Comment:
why like that? it looks weird and like a bug. Could you add comment?

Reply:
LOL I messed up something with that PR, let me reopen

Reply:
I messed up master an PR by accident and it showed the other way around, now it is proper, sorry :)